### PR TITLE
Fix file layout of ModSelectOverlay

### PR DESCRIPTION
**Pull request by peppy**
Usually wouldn't do this (waiting on an automated method to apply to all files), but this class is exceedingly misleading and hard to parse.

### CODE REVIEW

**Review by smoogipoo — APPROVED** [review]

**Comment by arphox** [comment]
Introducing automatically checked rules for this would definitely help us on this. It wouldn't be easy to introduce but would worth it in my opinion.

**Comment by peppy** [comment]
Yes, we are hoping to eventually standardise layout. See https://github.com/ppy/osu-framework/issues/2278.
